t/core_bools.t

```diff
@@ -0,0 +1,68 @@
+use strict;
+use warnings;
+use JSON::PP;
+use Test::More;
+BEGIN {
+  # this is only for JSON.pm
```

Comment:
This comment doesn't look right.

Reply:
It is correct. JSON.pm copies the tests from this dist and does regexes on the content. This is meant to make that easier.

Reply:
:vomiting_face: :shrug: